motor-control-module/src/main/java/org/sagebionetworks/research/motor_control_module/show_step_fragment/HandStepUIHelper.java

```diff
@@ -0,0 +1,66 @@
+package org.sagebionetworks.research.motor_control_module.show_step_fragment;
+
+import android.graphics.Matrix;
+import android.view.Display;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.sagebionetworks.research.domain.result.interfaces.TaskResult;
+import org.sagebionetworks.research.domain.task.Task;
+import org.sagebionetworks.research.mobile_ui.show_step.view.ShowUIStepFragmentBase;
+import org.sagebionetworks.research.mobile_ui.show_step.view.view_binding.UIStepViewBinding;
+import org.sagebionetworks.research.motor_control_module.step.HandStepHelper;
+import org.sagebionetworks.research.presentation.DisplayString;
+import org.sagebionetworks.research.presentation.model.interfaces.UIStepView;
+import org.sagebionetworks.research.presentation.show_step.show_step_view_models.ShowUIStepViewModel;
+
+public abstract class HandStepUIHelper {
```

Comment:
Why is this abstract?

Reply:
To prevent someone from instantiating one.

Reply:
A static util class idiom is to make a private no-arg constructor.

https://www.javaworld.com/article/2076040/core-java/effective-java.html